### PR TITLE
Update game-faq.md

### PR DESCRIPTION
**Pull request by Protogen187**
- Changed the wording in some sentences
- Removed "known issues" from title since there are no known issues posted in the file
- mentioned that a legit copy of BeamNG.Drive is needed for BeamMP to operate (since pirated copies are not supported)

### CODE REVIEW

File: docs/en/support/game-faq.md [hunk 1]
```diff
@@ -22,14 +22,13 @@ The Client is not officially  supported on Linux. However, you might be able to
 ---
 #### **Why is the launcher flagged by my antivirus or by Windows Defender**
 
-In order to prevent Windows Defender from flagging a specific .exe, we need to buy a license which is very expensive, therefore, BeamMP might get flagged by Windows Defender.
-
-Some antivirus may flag BeamMP as a virus  beacause it does some interactions with network and other stuff. There are **no viruses** in any of the code. The code for the launcher, server, and lua client can be seen on our [GitHub](https://github.com/BeamMP).
+For BeamMP to be a trusted source for various Antivirus programs, a very expensive licecence would be needed.
```
[the remark this answers is not in the document]
Comment (Protogen187) [reply docs/en/support/game-faq.md]: Thank you for pointing this out

File: docs/en/support/game-faq.md [hunk 2]
```diff
@@ -12,7 +12,7 @@ There is a full guide on how to install BeamMP on Windows, you can find it [here
 ---
 #### **Does BeamMP work on a cracked version of BeamNG?**
 
-BeamMP does not work with cracked versions or outdated versions of BeamNG.drive. If you want to use BeamMP please consider buying the game.
+BeamMP does not work with cracked versions or outdated versions of BeamNG.drive. If you want to use BeamMP, you need to own a legit copy of BeamNG.Drive off Steam.
```
[the remark this answers is not in the document]
Comment (Protogen187) [reply docs/en/support/game-faq.md:15]: I'd leave it at "legit"

[the remark this answers is not in the document]
Comment (Starystars67) [reply docs/en/support/game-faq.md:15]: Also "off steam" should be "from Steam".

[the remark this answers is not in the document]
Comment (Protogen187) [reply docs/en/support/game-faq.md]: I fixxed it

**Comment by Protogen187** [comment]
Cannot approve requested changes myself.
Need to wait for someone else to do so.

File: docs/en/support/game-faq.md [hunk 3]
```diff
@@ -22,14 +22,13 @@ The Client is not officially  supported on Linux. However, you might be able to
 ---
 #### **Why is the launcher flagged by my antivirus or by Windows Defender**
 
-In order to prevent Windows Defender from flagging a specific .exe, we need to buy a license which is very expensive, therefore, BeamMP might get flagged by Windows Defender.
-
-Some antivirus may flag BeamMP as a virus  beacause it does some interactions with network and other stuff. There are **no viruses** in any of the code. The code for the launcher, server, and lua client can be seen on our [GitHub](https://github.com/BeamMP).
+For BeamMP to be a trusted source for various Antivirus programs, a very expensive licence would be needed.
+Some antivirus prgroams may flag BeamMP as a threat, due to it interacting with the network and other stuff. There are **no viruses** in any of the code. The code for the launcher, server, and lua client can be found on our [GitHub](https://github.com/BeamMP).
```
[the remark this answers is not in the document]
Comment (ELKER54) [reply docs/en/support/game-faq.md:26]: programs not prgroams

**Comment by Protogen187** [comment]
@lionkor I made the changes

File: docs/en/support/game-faq.md [hunk 4]
```diff
@@ -12,7 +12,7 @@ There is a full guide on how to install BeamMP on Windows, you can find it [here
 ---
 #### **Does BeamMP work on a cracked version of BeamNG?**
 
-BeamMP does not work with cracked versions or outdated versions of BeamNG.drive. If you want to use BeamMP please consider buying the game.
+BeamMP is not guaranted not work with cracked versions or outdated versions of BeamNG.drive.
```
Comment (O1LER) [inline docs/en/support/game-faq.md:15]: `BeamMP is not guaranteed to work with`
or
`BeamMP may not work with`

File: docs/en/support/game-faq.md [hunk 5]
```diff
@@ -22,14 +22,12 @@ The Client is not officially  supported on Linux. However, you might be able to
...
+Some antivirus programs may flag BeamMP as a threat, due to it interacting with the network and other stuff. There are **no viruses** in any of the code. The code for the launcher, server, and lua client can be found on our [GitHub](https://github.com/BeamMP).
 
 ---
-#### **I have poor performance, what should I do?**
+#### **I have poor gameplay performance, what should I do?**
 
-We are working hard to make performance as good as possible. Sadly, there isn't a lot you can do about it, if you have already lowered your graphic settings and your performance is still poor, consider playing on a server with less players. The game is mainly CPU bound when you play with lots of people, so older CPUs (even quadcores) will suffer with more than a handful of people.
+We are working hard to make the Multiplayer experience as stable as possible. If you have already lowered your graphic settings and your performance is still poor, consider playing on a server with less players. The game is mainly CPU bound when you play with lots of people, so older CPUs (even quadcores) will suffer with more than a handful of people. (1 Car uses one CPU Thread)
```
Comment (O1LER) [inline docs/en/support/game-faq.md:30]: `One car uses one CPU thread`
or
`1 car uses 1 CPU thread`

also im not sure if we even want to include this. Its technically not correct, its more of a rule of thumb

**Review by O1LER — CHANGES_REQUESTED** [review]

File: docs/en/support/game-faq.md (hunk 5 above)
Comment (ELKER54) [reply docs/en/support/game-faq.md:30]: I would say leave it, we recommend it but its not a rule. Lots of people get good frames with lower core cpus, with high pops

**Review by Protogen187 — COMMENTED** [review]
grammar fix

**Review by O1LER — APPROVED** [review]

**Comment by Protogen187** [comment]
@lionkor Awaiting approval

File: docs/en/support/game-faq.md [hunk 6]
```diff
@@ -22,14 +22,12 @@ The Client is not officially  supported on Linux. However, you might be able to
 ---
 #### **Why is the launcher flagged by my antivirus or by Windows Defender**
 
-In order to prevent Windows Defender from flagging a specific .exe, we need to buy a license which is very expensive, therefore, BeamMP might get flagged by Windows Defender.
-
-Some antivirus may flag BeamMP as a virus  beacause it does some interactions with network and other stuff. There are **no viruses** in any of the code. The code for the launcher, server, and lua client can be seen on our [GitHub](https://github.com/BeamMP).
+Some antivirus programs may flag BeamMP as a threat, due to it interacting with the network and other stuff. There are **no viruses** in any of the code. The code for the launcher, server, and lua client can be found on our [GitHub](https://github.com/BeamMP).
```
Comment (Starystars67) [inline docs/en/support/game-faq.md:25]: Though correct the wording here feels off. 

Please rereview this bit.

**Review by ELKER54 — APPROVED** [review]
LGTM

File: docs/en/support/game-faq.md (hunk 6 above)
Comment (O1LER) [reply docs/en/support/game-faq.md:25]: Please open a new PR changing this, theres lots of other bits that need to be merged in order to enable translating

**Review by O1LER — APPROVED** [review]

**Pull request merged**